cmd/frontend/internal/usagestats/campaigns_test.go

```diff
@@ -0,0 +1,112 @@
+package usagestats
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/sourcegraph/sourcegraph/cmd/frontend/types"
+	"github.com/sourcegraph/sourcegraph/cmd/repo-updater/repos"
+	"github.com/sourcegraph/sourcegraph/internal/api"
+	"github.com/sourcegraph/sourcegraph/internal/db"
+	"github.com/sourcegraph/sourcegraph/internal/db/dbconn"
+	"github.com/sourcegraph/sourcegraph/internal/db/dbtesting"
+	"github.com/sourcegraph/sourcegraph/internal/extsvc"
+)
+
+func TestCampaignsUsageStatistics(t *testing.T) {
+	ctx := context.Background()
+	dbtesting.SetupGlobalTestDB(t)
+
+	// Create stub repo.
+	rstore := repos.NewDBStore(dbconn.Global, sql.TxOptions{})
+	now := time.Time{}
```

Comment:
If you use a zero value you don't need to use it in structs. So I think what you want is this:

```suggestion
	now := time.Now()
```